R/wwinference.R

```diff
@@ -62,21 +62,35 @@
 #'
 wwinference <- function(ww_data,
                         count_data,
-                        model_spec = wwinference::get_model_spec(
-                          forecast_date =
-                            "2023-12-06"
-                        ),
-                        mcmc_options = wwinference::get_mcmc_options(),
+                        model_spec = get_model_spec(),
```

Comment:
Ok this was a point I have a question on: should I just separate forecast date from the rest of the model specifications? It is the one thing that I really want the model to fail loudly on if a user doesn't provide a forecast date. So as is, if the user doesn't put a forecast date into `get_model_spec()` it will error with an informative error message. Should I just move forecast date out of the `get_model_spec()` function? I wasn't sure how to handle this

Reply:
> Should I just move forecast date out of the get_model_spec() function? I wasn't sure how to handle this

This is what I would do in the first instance.